crates/napi/src/threadsafe_function.rs

```diff
@@ -404,10 +418,9 @@ unsafe extern "C" fn call_js_cb<T: 'static, V: NapiRaw, R, ES>(
       sys::napi_get_and_clear_last_exception(raw_env, &mut error_result),
       sys::Status::napi_ok
     );
-    assert_eq!(
-      sys::napi_fatal_exception(raw_env, error_result),
-      sys::Status::napi_ok
-    );
+    // When shutting down, napi_fatal_exception sometimes returns another exception
+    let stat = sys::napi_fatal_exception(raw_env, error_result);
+    assert!(stat == sys::Status::napi_ok || stat == sys::Status::napi_pending_exception);
```

Comment:
I guess trying to raise an exception during shutdown doesn't make much sense, though? I could see why it might be like "hey it's too late for that, your exception failed". But it seems weird that it would be so intermittent. I really had to mash the refresh button a lot to get this one to happen 😂